src/middleware/passport-auth.ts

```diff
@@ -0,0 +1,168 @@
+import { createPrivateKey } from 'node:crypto';
+
+import passport from 'passport';
+import { Issuer, Strategy as OpenIdStrategy, TokenSet, UserinfoResponse } from 'openid-client';
+import { Strategy as GoogleStrategy } from 'passport-google-oauth20';
+import { Strategy as JWTStrategy, ExtractJwt } from 'passport-jwt';
+import { Repository } from 'typeorm';
+
+import { logger } from '../utils/logger';
+import { User } from '../entities/user';
+
+const readPrivateKey = (privateKey: string) => {
+    return createPrivateKey({ key: Buffer.from(privateKey, 'base64'), type: 'pkcs8', format: 'der' });
+};
+
+export const initPassport = async (userRepository: Repository<User>): Promise<void> => {
+    passport.use(
+        'jwt',
+        new JWTStrategy(
+            {
+                jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+                secretOrKey: process.env.JWT_SECRET!
```

Comment:
This can be a cert, but for now a secret will do.

Reply:
I agree... A secret will do for now.  We may want to look at certs again when we have more time.